docs/reference/controls/grid/sharedsizegroup.md

```diff
@@ -1,17 +1,17 @@
 ﻿---
 id: sharedsizegroup
-title: SharedSizeGroup
```

Comment:
This file shouldn't be translated, as it's not in zh-Hans subfolder. Can you revert it?

Reply:
Reverted, thanks for catching that!